ci-operator/config/openshift/cluster-api-provider-libvirt/master.yaml

```diff
@@ -6,6 +6,10 @@ base_images:
     tag: base
 binary_build_commands: make build INSTALL_DEPS=1
 canonical_go_repository: github.com/openshift/cluster-api-provider-libvirt
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: libvirt-machine-controller
```

Comment:
we should follow same convention here https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-provider-aws/master.yaml#L12